.github/workflows/deployment.yaml

```diff
@@ -193,6 +193,8 @@ jobs:
         run: |
           if ! az synapse notebook show --workspace-name phdi${TF_ENV}synapse${SHORT_CID} --name "Convert Parquet MPI"; then
             az synapse notebook create --workspace-name phdi${TF_ENV}synapse${SHORT_CID} --name "Convert Parquet MPI" --file "scripts/Synapse/convertParquetMPI.ipynb"
+          if ! az synapse notebook show --workspace-name phdi${TF_ENV}synapse${SHORT_CID} --name "Convert Parquet MCI"; then
```

Comment:
Oops, looks like this is missing a `fi` to close out the previous `if` statement before starting the next one!

Reply:
Good catch! Just added